Resources/views/MediaAdmin/list.html.twig

```diff
@@ -35,9 +35,11 @@ file that was distributed with this source code.
 {% endmacro %}
 
 {% block list_table %}
+    {% if datagrid.values['category'] is defined and datagrid.values['category']['value'] is defined %}
```

Comment:
Have you actually tested that? I'm asking because I believe `is defined` might act like `isset`, and here is how `isset` behaves : 

```
Interactive mode enabled

php > $a = [];
php > var_dump(isset($a['does']['not']['exist']));
bool(false)
```